LICENSE-BSD

```diff
@@ -1,3 +1,5 @@
+Copyright (c) 2015, Linaro Limited
```

Comment:
This mention doesn't properly reflect all the copyright owners, so I wonder what can be derived from this notice. I suppose it should be considered the default when a file doesn't have one.

Reply:
That's my understanding - the particular files and/or git history can be more definitive for a particular file or set of files.